openshift/templates/jobs/backups/backup-cron.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: batch/v1beta1
```

Comment:
I have faith in our developers that they can take this and re-use it. Any additional issues or troubleshooting is unlikely to have anything to do with the template...

Reply:
it would be much easier to re-use if they were already CLI consumable ala https://github.com/bcgov/gdx-agreements-tracker/blob/development/openshift/templates/postgres/deploy.yaml